### DISCUSSION TITLE
Create a subdirectory named modules leads to duplicate routes

### DISCUSSION THREAD

**Discussion by muuki88** [tgreyuk/typedoc-plugin-markdown#647] [Help] [answered]
Hi there :smiley: 

Thanks for this amazing plugin :pray: 

I'm currently stumbling over an issue where I have typescript API being generated in a sub folder named `modules`. However the plugin also generates an `index.md` and a `modules.md` with different contents. Because docusaurus treats both as the index page, it yields a duplicate routes error. See https://github.com/facebook/docusaurus/pull/9906/files

```
modules/
  index.md
  modules.md
```

`modules.md` contains the list of modules for which the API docs have been generated.

Question: Is there a way around this other then renaming the top-level folder?

**Comment by muuki88** [comment]
Cross post here as my attempt to provided a custom `isCategoryIndex` function doesn't let go of the warning.
https://github.com/facebook/docusaurus/issues/9780

**Comment by tgreyuk** [comment, marked as answer]
@muuki88  Hey - thanks. Have implemented a new option `modulesFileName` in typedoc-plugin-markdown@4.2.0. So you can name the page whatever you want:

```
"modulesFileName": "documentation"
```

**Reply by muuki88** [reply]
Wow 😲 thanks a lot
